organization_token.go

```diff
@@ -41,6 +41,7 @@ type OrganizationToken struct {
 	Description string    `jsonapi:"attr,description"`
 	LastUsedAt  time.Time `jsonapi:"attr,last-used-at,iso8601"`
 	Token       string    `jsonapi:"attr,token"`
+	ExpiredAt   time.Time `jsonapi:"attr,expired-at,iso8601"`
```

Comment:
This will allow functions which `Read` tokens to see their expiration times (b/c this is the object returned by the [read function](https://github.com/hashicorp/go-tfe/pull/672/files#diff-f74b71435c0aabb3498f3d2b393e5e62c756990e1fa72b8b0792b6c5a3b5941dR26)), but it is not going to allow for users of go-tfe to create tokens with an expiration.

For that we will need to update the [`Create` function](https://github.com/hashicorp/go-tfe/pull/672/files#diff-f74b71435c0aabb3498f3d2b393e5e62c756990e1fa72b8b0792b6c5a3b5941dR23) here to accept some kind of `OrganizationTokenCreateOptions` struct. Similar to how the user token creation works.

Reply:
Got it; will do! Thank you for the feedback 🙌

Reply:
Thanks for the call-out! It's true we'd like to maintain API compatibility for minor release versions. I think it would be a good idea to introduce CreateWithOptions in this case because it would not warrant a major version bump. We can always combine the two when we do rev the major version...

Speaking of compatibility, I also want to point out that the default ExpiredAt value for TFE versions that don't have this capability yet will appear in the past (January 1, year 1, 00:00:00.000000000 UTC). Not a huge deal, because callers can use ExpiresAt.IsZero() I suppose, and it should be very rare in practice.

Reply:
Good to know! Thanks so much for the feedback, @brandonc!